arm/.global/global.module.tests.ps1

```diff
@@ -246,15 +246,15 @@ Describe 'Readme tests' -Tag Readme {
             ($ComparisonFlag -gt 2) | Should -Be $false
         }
 
-        It '[<moduleFolderName>] parameters section should contain all parameters from the deploy.json file' -TestCases $readmeFolderTestCases {
+        It '[<moduleFolderName>] parameters section should contain all parameters from the template file' -TestCases $readmeFolderTestCases {
             param(
                 $moduleFolderName,
                 $templateContent,
                 $readMeContent
             )
 
             $ReadmeHTML = ($readMeContent | ConvertFrom-Markdown -ErrorAction SilentlyContinue).Html
-            $parameters = Get-Member -InputObject $templateContent.parameters -MemberType NoteProperty
+            $parameters = $templateContent.parameters.Keys
```

Comment:
Minor: should we align to use either 'Keys' or 'keys' throughout the script?

Reply:
Keys it is